### PR TITLE
Upgrade warning

### PR DESCRIPTION
**Pull request by Ener-Getick**
ping @xabbuh @lsmith77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ applications with Symfony. Features include:
 [![Latest Stable Version](https://poser.pugx.org/FriendsOfSymfony/rest-bundle/v/stable.svg)](https://packagist.org/packages/FriendsOfSymfony/rest-bundle)
 [![SensioLabsInsight](https://insight.sensiolabs.com/projects/0be23389-2e85-49cf-b333-caaa36d11c62/mini.png)](https://insight.sensiolabs.com/projects/0be23389-2e85-49cf-b333-caaa36d11c62)
 
+Note
+----
+
+FOSRestBundle 1.x is not longer maintained, 1.8 only receives security fixes. Please upgrade to FOSRestBundle 2.x as soon as possible.
```
Comment (xabbuh) [inline README.md]: no longer

**Comment by xabbuh** [comment]
👍 after the small typo fix

**Comment by Ener-Getick** [comment]
I can't access github right now :/ (dns issues i guess). I'll try to fix it
tomorrow.

**Comment by xabbuh** [comment]
@Ener-Getick Not sure what you mean. What do you wanna fix?

**Comment by Ener-Getick** [comment]
@xabbuh nothing, i sent this message by mail a while ago and it looks like github posted it only now. Sorry for the inconvenience.

**Pull request merged**